libtelemetry/src/full/java/com/mapbox/android/telemetry/MapboxTelemetry.java

```diff
@@ -37,7 +39,6 @@
   private static final String LOG_TAG = "MapboxTelemetry";
   private static final String NON_NULL_APPLICATION_CONTEXT_REQUIRED = "Non-null application context required.";
   private static AtomicReference<String> sAccessToken = new AtomicReference<>("");
```

Comment:
why the `s` in front? Better way to name this variable?

Reply:
Thats right. It does not seem to be following the convention. Changing it to accessToken.